### PR TITLE
Update packages' recipes to their latest versions from litex-conda-packages

### PR DESCRIPTION
**Pull request by ajelinski**

### CODE REVIEW

**Comment by mithro** [comment]
This appears to be failing on travis.

**Comment by ajelinski** [comment]
> This appears to be failing on travis.

`libftdi` was failing and the next stage didn't even started because of it. I didn't notice it has been disabled in `litex-conda-packages` from building on Linux for a few months now. Such changes fix the problem with conflicts on Linux: https://github.com/litex-hub/litex-conda-prog/pull/1/commits/c1e3332f8a8cceaeb2e325975ce5719ea8b22a7f .

I tested the same recipe changes also on Windows and macOS and the package still builds successfully there.

**Comment by ajelinski** [comment]
Trying to fix the problem I tested using newer `libftdi` source files (v1.4 vs v1.3 used now) and it didn't solve the problem but `libftdi` v1.4 can be build successfully after applying changes from https://github.com/litex-hub/litex-conda-prog/commit/c1e3332f8a8cceaeb2e325975ce5719ea8b22a7f. Would you like me to add an additional commit updating `libftdi` to v1.4?

File: prog/iceprog/meta.yaml [hunk 1]
```diff
@@ -23,7 +24,13 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - pkg-config
+    - pkg-config [not win]
```
Comment (mithro) [inline prog/iceprog/meta.yaml:27]: Can you align these tags and group them? (Please use spaces)

**Pull request merged**